types/markdown-it-anchor/markdown-it-anchor-tests.ts

```diff
@@ -0,0 +1,21 @@
+import * as MarkdownIt from "markdown-it";
+import {anchor} from "markdown-it-anchor";
```

Comment:
Like my comment on #15884, this library uses [`module.exports = anchor`](https://github.com/valeriangalliat/markdown-it-anchor/blob/4062d08dd67eb83d2e41d5269bddba51c51d601c/index.js).